pkg/kubelet/apis/cri/runtime/v1alpha3/api.proto

```diff
@@ -1107,6 +1116,40 @@ message StatusResponse {
     map<string, string> info = 2;
 }
 
+// LinuxIDMapping represents a single user namespace mapping in Linux.
+message LinuxIDMapping {
+   // container_id is the starting id for the mapping inside the container.
+   uint32 container_id = 1;
+   // host_id is the starting id for the mapping on the host.
+   uint32 host_id = 2;
+   // size is the length of the mapping.
+   uint32 size = 3;
+}
+
+// LinuxUserNamespaceConfig represents runtime's user-namespace configuration on a linux host.
+message LinuxUserNamespaceConfig {
+   // uid_mappings is an array of user id mappings.
+   repeated LinuxIDMapping uid_mappings = 1;
+   // gid_mappings is an array of group id mappings.
+   repeated LinuxIDMapping gid_mappings = 2;
+}
+
+// GetRuntimeConfig are the configuration details from the runtime.
+message GetRuntimeConfig {
+    // LinuxUserNamespaceConfig represents runtime's user-namespace configuration on a linux host.
+    LinuxUserNamespaceConfig user_namespace_config = 1;
```

Comment:
Is there any reason not to append the field to `message RuntimeConfig` instead? Also `GetRuntimeConfig` seems to be slightly odd name for the type.

Reply:
I still think we should try to find a name that doesn't involve `Get` as a prefix :-)

Reply:
done.